documentation/modules/retrieving-vmware-mor.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="retrieving-vmware-mor_{context}"]
+= Retrieving a VMware vSphere MOR
+
+When you migrate VMs with a VMware vSphere source provider using {project-first} from the CLI, you need to know the managed object reference (MOR) of certain entities in vSphere, such as datastores, networks and VMs.
```

Comment:
Remember to use the Oxford comma in comma separated lists
```suggestion
When you migrate VMs with a VMware vSphere source provider using {project-first} from the CLI, you need to know the managed object reference (MOR) of certain entities in vSphere, such as datastores, networks, and VMs.
```

Reply:
Doh! 

Note that Vale didn't catch that... [still on me, though] 
![image](https://github.com/kubev2v/forklift-documentation/assets/60698649/e220ad31-7522-45a4-9a2e-50ee8e14721d)

Reply:
NOTE: Vale is a linter for RH style. 
If you need a grammar linter to correct basic grammar errors, I am happy to suggest one to you.